src/data/faq.json

```
@@ -818,7 +835,7 @@
                             "Make sure that exposure logging is active.",
                             "If the latest version still shows the issue, it is probably due to individual power saving modes or battery settings by different manufacturers. These do not fully comply with the Android specifications and often apply aggressive power saving measures. <b>Corona-Warn-App is affected by these power saving settings and therefore often cannot update the risk status in the background for a long time. However, you can change the settings in the Corona-Warn-App:</b>",
                             "In order to activate Prioritized Background Activity you need to de-activate optimization of the battery usage:",
-                            "<ol><li>In the Corona-Warn-App (CWA) on the Start Screen choose the three-dot icon in the top right corner, then Settings</li><li>Choose Prioritized Background Activity (If Prioritized Background Activity is 'On' already, you don't need to take further action here.)</li><li>Choose 'OPEN DEVICE SETTINGS'</li><li>You see now the operating system 'Optimize battery usage' / 'Battery optimization' or similar (depends on device model). Choose in the dropdown list 'All' or 'All Apps'</li><li>Scroll down to Corona-Warn</li><li>Move the slider to the left (i.e. optimize battery usage off) or tap on the entry and then choose 'Do not optimize' (This procedure depends on the device model.)</li><li>Cross-check the change in the CWA:</li><li>In the Corona-Warn-App (CWA) on the Start Screen choose the three-dot icon, then Settings</li><li>Prioritized Background Activity should be 'On' now.</li></ol>",
+                            "<ol><li>In the Corona-Warn-App (CWA) on the Start Screen choose the three-dot icon in the top right corner, then Settings</li><li>Choose Prioritized Background Activity (If Prioritized Background Activity is 'On' already, you don't need to take further action here.)</li><li>Choose 'OPEN DEVICE SETTINGS'</li><li>You see now the operating system 'Optimize battery usage' / 'Battery optimization' or similar (depends on device model). Choose in the dropdown list 'All' or 'All Apps'</li><li>Scroll down to Corona-Warn</li><li>Move the slider to the left (i.&nbsp;e. optimize battery usage off) or tap on the entry and then choose 'Do not optimize' (This procedure depends on the device model.)</li><li>Cross-check the change in the CWA:</li><li>In the Corona-Warn-App (CWA) on the Start Screen choose the three-dot icon, then Settings</li><li>Prioritized Background Activity should be 'On' now.</li></ol>",

```

```suggestion
                            "<ol><li>In the Corona-Warn-App (CWA) on the Start Screen choose the three-dot icon in the top right corner, then Settings</li><li>Choose Prioritized Background Activity (If Prioritized Background Activity is 'On' already, you don't need to take further action here.)</li><li>Choose 'OPEN DEVICE SETTINGS'</li><li>You see now the operating system 'Optimize battery usage' / 'Battery optimization' or similar (depends on device model). Choose in the dropdown list 'All' or 'All Apps'</li><li>Scroll down to Corona-Warn</li><li>Move the slider to the left (i.e. optimize battery usage off) or tap on the entry and then choose 'Do not optimize' (This procedure depends on the device model.)</li><li>Cross-check the change in the CWA:</li><li>In the Corona-Warn-App (CWA) on the Start Screen choose the three-dot icon, then Settings</li><li>Prioritized Background Activity should be 'On' now.</li></ol>",
```
Remove `&nbsp; `from English abbreviation.